zuul.d/kuttl.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- job:
+    name: cifmw-base-kuttl
+    nodeset: centos-9-crc-xxl
+    timeout: 5400
+    abstract: true
+    vars:
+      crc_parameters: "--memory 20000 --disk-size 120 --cpus 6"
+    parent: base-crc
+    pre-run:
+      - ci/playbooks/e2e-prepare.yml
+      - ci/playbooks/dump_zuul_vars.yml
+    run:
+      - ci/playbooks/kuttl/run.yml
+    post-run: ci/playbooks/collect-logs.yml
+
+- job:
+    name: cifmw-kuttl
+    parent: cifmw-base-kuttl
+    files:
+      - ^ci/playbooks/kuttl/.*
+      - ^scenarios/centos-9/kuttl.yml
+      - ^zuul.d/kuttl.yaml
+    vars:
+      cifmw_kuttl_tests_operator_list:
```

Comment:
Any specific criteria for this list of operators? If it's meant to be exhaustive, there are a few other operators that have kuttl tests

Reply:
I tried all the operators and this list seems to be passing that's why i have added it here. I will take a look at the failing one as a follow up.